.ci.yaml

```diff
@@ -402,6 +402,7 @@ targets:
           {"dependency": "clang"},
           {"dependency": "cmake"},
           {"dependency": "ninja"}
+          {"dependency": "android_sdk", "version": "version:29.0"},
```

Comment:
ci.yaml looks good to me. Please ping me when you land this as infra will need to manually roll this to flutter/infra as the bot is broken (https://github.com/flutter/flutter/issues/93225)